lib/openh264.py

```diff
@@ -44,7 +44,8 @@ def EncodeCommandLine(self, parameters, bitrate, videofile, encodedfile):
         '-rc 0 -maxbrTotal 0 -tarb %d '
         '-ltarb 0 %d '
         '-lmaxb 0 0 '
-        '-frout 0 %d ' #this is to set the output frame rate, TODO: add option of input frame rate after the codec supports it
+        '-numtl 1 '
```

Comment:
Yes it is Temporal layer number, the current openh264.cfg is using 2 which will replace the default 1. Don't need to set it to be 2 in performance tests.